.golangci.yml

```diff
@@ -12,13 +12,15 @@ linters-settings:
     # put imports beginning with prefix after 3rd-party packages;
     # it's a comma-separated list of prefixes
     local-prefixes: github.com/cmsgov/easi-app
-  golint:
-    min-confidence: 0
+  revive:
+    confidence: 0
+  gosec:
+    excludes:
+      - G404
```

Comment:
Why disable G404?

Reply:
Looks like this is warning about the use of `math/rand` instead of `crypto/rand` to generate random numbers. It's probably worth marking any lines where we're doing that with a specific `// #nosec G494` instead fo dialing it globally, though.

Reply:
It's not trivial, crypto doesn't have a "int between 1 and n" function